airflow/www_rbac/api/experimental/endpoints.py

```diff
@@ -257,6 +258,40 @@ def dag_run_status(dag_id, execution_date):
     return jsonify(info)
 
 
+@csrf.exempt
+@api_experimental.route('/dags/<string:dag_id>/cancel_dag_run', methods=['POST'])
+@requires_authentication
+def cancel_dag_run(dag_id):
+    """
+    Cancel a specific dagrun run_id for a given DAG
```

Comment:
What's the difference between this endpoint and the one above? Is that for if you aren't using an RBAC setup?

Reply:
one is for rbac flask app and the other is for the non-rbac flask app (which we don't use but the version of airflow we are on does still support.